std/traits.d

```diff
@@ -6698,6 +6704,22 @@ unittest
     static assert(getUDAs!(c, Attr)[0].value == 42);
     static assert(getUDAs!(c, Attr)[1].name == "Pi");
     static assert(getUDAs!(c, Attr)[1].value == 3);
+
+    struct AttrT(T)
+    {
+        string name;
+        int value;
```

Comment:
For the correctness of the unit test? No. For clarity? Yes. I'd prefer not having to explain that the "T" in this struct declaration has no purpose.